File: PWRSyllabus.Core/Entities/Employee.cs
using System.Collections.Generic;

namespace PWRSyllabus.Core.Entities
{
    public class Employee : BaseEntity
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string EmailAddress { get; set; }
        public List<SubjectCard> Subjects { get; set; }
    }
}

File: PWRSyllabus.API/ClientApp/src/models/Employee.ts
export default interface Employee {
    id: number;
    firstName: string;
    lastName: string;
    emailAddress: string;
}

const DefaultEmployee: Employee = {
    id: 0,
    firstName: 'Jerzy',
    lastName: 'Pietruszko',
    emailAddress: 'jerzy.pietruszko@pwr.edu.pl',
};

export {Employee, DefaultEmployee};